app/helpers/users_helper.rb

```diff
@@ -1,2 +1,7 @@
 module UsersHelper
+  def gravatar_for user, size: 80
```

Comment:
Please use the config gem to manage the constants. 
For example, like the way you used it in the User model.